packages/web-frontend/src/containers/DashboardGroup/index.js

```diff
@@ -51,9 +51,10 @@ export default class DashboardGroup extends Component {
     }, []);
     // Filter out views that have the same viewId (removes duplicates across subTabs)
     // Note this is really inefficient, effectively a loop within a loop. O(n^2).
-    const uniqueViews = allViews.filter(
-      (view, index, arr) => arr.findIndex(v => v.viewId === view.viewId) === index,
-    );
+    const uniqueViews = allViews
+      .filter(view => !view.drillDownLevel)
```

Comment:
Filter out the drillDown reports so that they are not rendered with the rest of the dashboard reports